### DISCUSSION TITLE
s3.to_parquet: append mode

### DISCUSSION THREAD

**Discussion by raestadilla** [aws/aws-sdk-pandas#1120] [Q&A] [answered]
Hi, I've been trying the s3.to_parquet function with mode = "append". I know that we cannot directly update data/tables in s3/athena but the s3.to_parquet documentation says that mode = append will UPSERT to an existing table. But based on my trials, it only appends data and doesn't perform updates. Any thoughts? Thanks!

Here's my code sample:
`wr.s3.to_parquet(
    df=new_df,
    database="my_database",
    table="sample_table",
    dataset=True,
    mode="append",
    path=s3_path
)`

Documentation:
"On append mode, the parameters will be upsert on an existing table." - https://aws-data-wrangler.readthedocs.io/en/2.13.0/stubs/awswrangler.s3.to_parquet.html#awswrangler.s3.to_parquet

**Comment by jaidisido** [comment, marked as answer]
Hi @raestadilla, I believe that part of the docs references how the `parameters` of the Glue table would be modified as a result of an append with new columns. It's confusing though so will clarify the documentation. Thanks
